experimenter/experimenter/features/configs/apps.yaml

```
@@ -5,16 +5,20 @@
 fenix:
   repo: @mozilla-mobile/firefox-android
   fml_path: fenix/app/nimbus.fml.yaml
-  branch_mapping: "v{version}"
+  major_release_branch: "releases_v{major}"
+  minor_release_tag: "fenix-v{major}.{minor}"
```

Fenix just has `v`, not the app name like focus does

```suggestion
  minor_release_tag: "v{major}.{minor}"
```